.krew.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     {{addURIAndSha "https://github.com/rook/kubectl-rook-ceph/archive/{{ .TagName }}.zip" .TagName }}
     files:
     - from: "kubectl-rook-ceph-*/kubectl-rook-ceph.sh"
-      to: "*"
```

Comment:
Let's make this the commit title: `fix small typos in doc and config yaml`. It's better to have a good title and no description than to leave all the details in a 1-line description.